glooshot/docs/tutorial/bookinfo_tutorial.md

```diff
@@ -0,0 +1,265 @@
+```
+
+- We will install the bookinfo app in the default namespace. Let's first label it for autoinjection
+  - This allows Istio to interface with our app.
+```bash
+kubectl label namespace default istio-injection=enabled
+```
+
+#### Provide metric source configuration to Prometheus
+
+Prometheus is a powerful tool for aggregating metrics. To use Prometheus most effectively, you need to tell it where it
+can find metrics by specifying a list of [scrape configs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
+
+Here is an [example config](https://github.com/morvencao/istio/blob/036f689ae211cd320d68412eb42916d2debb1b73/install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml#L15) for how Istio's metrics should be handled by Prometheus.
+As you can see, scrape configs that are both insightful and resource-efficient can be quite complicated.
+Additionally, managing Prometheus configs for multiple scrape targets can be difficult.
+
+Fortunately, Supergloo provides a powerful utility for configuring your Prometheus instance in such a way that is
+appropriate for your chosen service mesh.
+
+By default, `glooshot init` deploys an instance of Prometheus (this can be disabled).
+For best results, you should configure this instance of Prometheus with the metrics that are relevant to your particular service mesh.
+We will use the `supergloo set mesh stats` utility for this.
+
+```bash
+supergloo set mesh stats \
+    --target-mesh glooshot.istio-istio-system \
+    --prometheus-configmap glooshot.glooshot-prometheus-server
+```
+
+Note that we just had to tell Supergloo where to find the mesh description and where to find the config map that we want to update.
+Supergloo knows which metrics are appropriate for the target mesh and sets these on the active prometheus config map.
+You can find more details on setting Prometheus configurations with Supergloo [here](https://supergloo.solo.io/tutorials/istio/tutorials-3-prometheus-metrics/).
+
+#### Deploy the bookinfo app
+
+- Now deploy the bookinfo app to the default namespace
+```bash
+kubectl apply -f https://raw.githubusercontent.com/solo-io/glooshot/master/examples/bookinfo/bookinfo.yaml
+```
+
+- Verify that the app is ready.
+  - When the pods in the `default` namespace are ready, we can start testing our app
+```bash
+kubectl get pods -n default -w
+```
+
+- Let's access the landing page of our app
+  - Execute the command below.
+  - Navigate to http://localhost:9080/productpage?u=normal in your browser.
+  - You should see a book description, reviews, and ratings - each provided by their respective services.
+  - Reload the page a few times, notice that the ratings section changes. Sometimes there are no stars, other times red or black stars appear. This is because Istio is load balancing across the four versions of the reviews service. Each reviews service renders the ratings data in a slightly different way.
+```bash
```

Comment:
For some reason the formatting here doesn't look right.. it could just be needing an empty line before the ``` marks